index.json

```diff
@@ -1056,7 +1056,7 @@
     },
     "nightly": {
       "url": "https://wicg.github.io/frame-timing/",
-      "repository": "https://github.com/w3c/frame-timing"
+      "repository": "https://github.com/wicg/frame-timing"
```

Comment:
ha, this uncovered another bug - I guess in my check of repo existence, I should have checked they were not archived either